packages/pwa-kit-extension-sdk/README.md

```diff
@@ -107,6 +107,12 @@ Now when the base application is built the import for the `logo.ts` in `extensio
 
 This mechanism is useful when you want to allow for fine grained customization of your application extension. Its the responsibility of the extension developer to ensure their document what files are overridable and what the expected input and output of those files are. For example, if the overridable file is a React component you should document the props that get passed to that component and the expected exports of the file.
 
+#### Forced overrides
+Sometimes, you may need to override some files from an extension but the extension developers do not allow those files to be overridden, you can use `forced_overrides` to bypass these restrictions.
```

Comment:
🤔 I'm not sure whether this is the right place to describe forced overrides. Because at this point, the readme is talking about `The anatomy of an Application Extension` (see line 25). But forced overrides is a feature that is used in the base project (not in the app extension).

So what is the correct place? I'm thinking `template-typescript-minimal/README.md` file. But that readme does not have any existing info on app extensibility.

Reply:
Typescript minimal is a private project atm. I asked the same question previously too, Adam advised we can put it here to explain about overrides machenism since the logic stays in this package. I think it is okay since we talked about overrides in this section. it is logical we can mentioned about force_overrides too

Reply:
Even though typescript-minimal is a private package, its readme will be part of the generated project. So other folks will still be able to read it in their project.

Reply:
If we still want to mention forced overrides in this file, let's rewrite it a bit, so that it makes more sense to the audience in this case, which are the extension developers. Perhaps something like this:

```
There may be cases where the users of your extension may want to override some of your files that you have not allowed to be extended. In this case, please know that there exists `forced_overrides` for them use to bypass these restrictions...
```